tests/album-create-test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-undef */
```

Comment:
You can remove the need for this by telling eslint you are using jest,  you can do that by adding `"jest": true` into the "env" object of your `.eslintrc.json` file
Once done you can remove that comment from the top of all your test files